lib_comfyui/webui_tab.py

```diff
@@ -61,12 +61,13 @@ def can_install_at(path):
 It looks like your ComfyUI installation isn't set up yet!  
 Go to `Settings > ComfyUI`, and set the proper install location.  
 
-Alternatively, if you don't have a ComfyUI install yet, you can (literally) just clic this button to install it automatically: 
+Alternatively, if you don't have a ComfyUI install yet, you can just click this button to install it automatically: 
```

Comment:
hey è.é

Reply:
I think you should change it to 2 sentences if you wanna remove the cool "familiar" way of talking of the sentence:
"... you can click this button. It will install ComfyUI automatically."

Reply:
Sorry. I can revert back. It just seemed a little too trying to be impressive for a button and I changed it on a whim.

Reply:
Well I think it's not that big of a deal as users will only see it for a short period. 

I don't mind chamging it, just that if we do, we should totally make it 2 sentences on 2 lines imo.

Reply:
another 5th one:
![Screenshot_20230321-105623](https://user-images.githubusercontent.com/34081873/226646689-7f77aa45-8d72-4da7-9bd9-91838d9f3386.png)

Reply:
chose the one you prefer lol

I really like "Haven't installed ComfyUI yet? No problem! Just click this button and it will be installed automatically:"

Reply:
Should we add a red button to uninstall ComfyUI in the settings? When you click on it, an additionnal pop-up pops up to ask "are you sure"?

Reply:
Hm may be unnecessary tbh. Just go to folder and delete. We have other priorities. I like your suggestion "Haven't installed ComfyUI yet? No problem! Just click this button and it will be installed automatically:", putting it in.

Reply:
Sorry marked as resolved but is the updated text good @John-WL?

Reply:
Sorry marked as resolved but is the updated text good @John-WL?